### DISCUSSION TITLE
I'm usually not easily impressed, but...

### DISCUSSION THREAD

**Discussion by kyr0** [GrapesJS/grapesjs#4875] [Show and tell]
...thank you @artf! This is one of the very few projects and codebases that I discovered over the course of 20+ years of my professional development career where I'm absolutely impressed by the implementation, the documentation and the community support at the same time.

I wish you the greatest success with this project. I was about to implement a WYSIWYG editor from scratch and was planning to spend a good amount of time on that but you saved me here and I couldn't wish for a better API. Just wow.

**Comment by artf** [comment]
Thank you Aron, I really appreciate your kind words, that mean a lot 🙏 ❤️
